lessons/16-css-site/contact.html

```diff
@@ -0,0 +1,42 @@
+<html>
+    <head>
+        <title>About | Robert Stewart</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <header>
+        <img src="https://scontent.fyyc3-1.fna.fbcdn.net/v/t1.0-9/16105889_10154981078039626_1168822799549117995_n.jpg?_nc_cat=104&_nc_oc=AQkwgXRrtLMdPW1Iqns3SCM4_wxBNOrwG641UOKNbBq_8tCFQILkRNA3yCcX1JoBkrcup6uwmm0NAbhOZ86R6JWS&_nc_ht=scontent.fyyc3-1.fna&oh=38df1481904139d8483be8ea6abaeb9c&oe=5E2F1692" width="300" height="300" alt="Robert Stewart">
+        <h2>Robert Stewart's Portfolio</h2>
+        <nav>
+            <a href="contact.html" class="active">Contact</a>
+            
+            <a href="about.html" >About</a>
+            
+            <a href="index.html" >Home</a>
+        </nav>
+    </header><hr>
+    <main>
+        <h1>Contact</h1>
+        <fieldset>
+            <legend>Contact Me!</legend>
+
+            <form action="contact-success.html" method="GET">
+                <div> 
+                    <label for="Name">
+                        <input type="text" name="name" id="name" maxlength="30" placeholder="First and last name" required>
+                    </label>
+
+                    <label for="Email">
+                        <input type="email" name="email" id="email" placeholder="Email" required>
+                    </label>
+                
+                </div> 
+                    <label for="Message">
+                        <textarea name="message" id="message" cols="30" rows="10" placeholder="Message to send Bobby Stewart" required></textarea>
+                    </label>    
+                <div>
+                    <input type="submit" value="Send Email" id="submit">
+                </div>
+            </form>
+        </fieldset>

```

Comment:
Everything looks great! The only suggestion I'd make is to swap the `<fieldset>` and `<form>` tags, so that the fieldset is inside of the form